### PR TITLE
win_scheduled_task module for windows

### PR DESCRIPTION
**Pull request by petemounce**
Fledgling module to allow scheduled tasks to be managed.

Note: At present, I only need enabled/disabled support. I'm using it to disable windows update scheduled tasks, since I'm managing that process myself. 

There's lots of scope for more features. See `Get-Help *Scheduled*` for inspiration :)

### CODE REVIEW

**Comment by bcoca** [comment]
@cchurch

File: windows/win_scheduled_task.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENTATION = '''
+---
+module: win_scheduled_task
+version_added: "1.9"
```
Comment (resmo) [inline windows/win_scheduled_task.py]: update to "2.0"

File: windows/win_scheduled_task.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+short_description: Manage scheduled tasks
+description:
+    - Manage scheduled tasks
+options:
+  name:
+    description:
+      - Name of the scheduled task
+      - Supports * as wildcard
+    required: true
+    default: null
```
Comment (resmo) [inline windows/win_scheduled_task.py]: if required, there is no default. should be removed.

File: windows/win_scheduled_task.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    aliases: []
```
Comment (resmo) [inline windows/win_scheduled_task.py]: if there aren't aliases, no need to add a empty list. could be removed

**Comment by petemounce** [comment]
@resmo thanks for the feedback; done.

**Comment by gregdek** [comment]
Adding @owner_pr flag because @petemounce is co-owner of other Extras modules

**Comment by bcoca** [comment]
instead of state=enabled|disabled, a enabled=yes|no flag is more within ansible norm. Also, could the module create/delete scheduled tasks, then state=present|absent will also be nice to have.

**Comment by petemounce** [comment]
@bcoca I agree that adding create/delete would be a nice to have - but I don't need it, currently. It is a significant chunk of work that I won't have time to do in the short term.

**Comment by perzizzle** [comment]
Has anyone signed up to add create/delete to this module? I have the need and could spend a little time working on it.

**Comment by petemounce** [comment]
@perzizzle I have not, but happy to test those changes.

My particular use case for this module is disabling Windows update related tasks before baking an image, so I don't get punched with them running later and interesting with production traffic. Those tasks afaik can only be disabled, not deleted.

**Comment by gregdek** [comment]
@petemounce is this ready for re-review in your opinion?

**Comment by petemounce** [comment]
@gregdek yes. I have addressed all the points. I am not planning on increasing the scope to allow create/delete of scheduled tasks as @bcoca suggested - if that's required, then this should either not get merged, or get merged and an issue opened to come back and add it (@perzizzle was interested to implement).

**Comment by perzizzle** [comment]
I'm still interested in adding this feature it just turned out to be non-trivial. I can have something ready within in the next week or two.

**Comment by gregdek** [comment]
Thanks @perzizzle.

**Comment by bcoca** [comment]
Ok with merging as is now and trying to add more features later on, incremental is good.

**Comment by petemounce** [comment]
Thanks for the merge! :)

**Pull request merged**